package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "lab-01-node-ecosystem",
+  "version": "1.0.0",
+  "description": "![CF](https://camo.githubusercontent.com/70edab54bba80edb7493cad3135e9606781cbb6b/687474703a2f2f692e696d6775722e636f6d2f377635415363382e706e67) 401 JS -- Class 01 node ecosystem  ===",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "expect.js": "^0.3.1"
```

Comment:
looks like you forgot to install mocha. your test won't run without installing mocha.